.github/CONTRIBUTING.md

```diff
@@ -6,9 +6,10 @@ We accept pull requests for bug fixes and features where we've discussed the app
 
 Please do:
 
-* Check existing issues to verify that the [bug][bug issues] or [feature request][feature request issues] has not already been submitted.
+* Check existing issues to verify that an existing [bug][bug issues] or [feature request][feature request issues] issue does not already exist for the same problem or feature.
```

Comment:
Too many "exist(ing)": "existing issues", "an existing bug or fr", and "does not already exist".

How about

* Check issues to verify that a [bug][bug issues] or [feature request][feature request issues] issue does not already exist for the same problem or feature.

Reply:
👋 @muzimuzhi that's good feedback. Would you be interested in opening a PR - I'd be happy to review it

Reply:
Done. See #9772.